packages/experimental/RadioGroup/src/Radio/useRadio.ts

```diff
@@ -104,6 +104,7 @@ export const useRadio = (props: RadioProps): RadioInfo => {
       focusable: !state.disabled,
       enableFocusRing: enableFocusRing ?? true,
       onAccessibilityAction: onAccessibilityAction,
+      disabled: state.disabled,
```

Comment:
Disabled should also be accessible from radio.props (and hence mergedProps). Not having it here can lead to unintended bugs, even if disabled is passed as prop, it will be listed as undefined in the radio.props!
The disabled radio still being selectable was also due to this, disabled was not getting applied to root pressable since it was present only in the state and not in the mergedProps.